modules/dkan/dkan_datastore/src/Helper/Node.php

```diff
@@ -0,0 +1,35 @@
+<?php
+namespace Dkan\Datastore\Helper;
+
+class Node {
+  /**
+   * Gets nid using uuid.
+   */
+  public static function getNid($uuid) {
+    $nid = db_query('SELECT nid FROM {node} WHERE uuid = :uuid', array(':uuid' => $uuid))->fetchField();
+    if ($nid) {
+      return $nid;
+    }
+    else {
+      return FALSE;
+    }
+  }
+
+  /**
+   * Loads node from uuuid.
+   */
+  public static function getNodeFromUuid($uuid) {
+    static $nodes = array();
+    if (!isset($node[$uuid])) {
```

Comment:
Should this be $nodes, and can you not init $nodes to array() or it would get wiped out?

Reply:
you are correct, it should be nodes. the static keyword makes the $nodes variable persistent across calls to this function in the current process, so after it is initialized, the first time, it will be available to future calls of the function without getting overridden.